mindsdb/integrations/handlers/postgres_handler/postgres_handler.py

```diff
@@ -549,3 +551,215 @@ def process_event(event):
             conn.commit()
 
             conn.close()
+
+    def meta_get_tables(self, table_names: Optional[list] = None) -> Response:
```

Comment:
Should it also collect tables from not default schema? Postgres handler will have access to all schema that defined in db (if there are no user restrictions). Agent also can work with schema names.

Reply:
Hmm.. That is true. @ZoranPandovski What do you think? Do we want to do this?